TYPO3.Flow/Classes/TYPO3/Flow/I18n/Parser/DatetimeParser.php

```diff
@@ -163,7 +167,7 @@ protected function doParsingInStrictMode($datetimeToParse, array $parsedFormat,
             foreach ($parsedFormat as $subformat) {
                 if (is_array($subformat)) {
                     // This is literal string, should match exactly
-                    if (\TYPO3\Flow\I18n\Utility::stringBeginsWith($datetimeToParse, $subformat[0])) {
+                    if (Utility::stringBeginsWith($datetimeToParse, $subformat[0])) {
```

Comment:
Also suboptimal, but I guess `Utility::stringBeginsWith` is still self-explaining enough.